Example/src/DetachedHeader.tsx

```diff
@@ -49,8 +50,39 @@ DetachedHeader.Container = ({
   children: JSX.Element;
 }) => {
   return (
-    <View style={{ position: 'absolute', top: 0, left: 0, right: 0 }}>
+    <View
+      style={{
+        position: 'absolute',
+        top: 0,
+        left: 0,
+        right: 0,
+      }}
+    >
       {children}
     </View>
   );
 };
+
+DetachedHeader.AnimatedContainer = ({
```

Comment:
I would just merge it with the previous one

Reply:
Got it